webanno/webanno
# #1753 - Allow editing constraints in the settings UI

**What's in the PR**
- Add option to create constraints set in the UI
- Made constraints editable in the UI

**How to test manually**
* Create a new constraints set directly in the UI
* Edit it
* Save it
* Upload a set and edit it
* Try saving a bad rules set, e.g. one simply containing the text `This is a test` - it should give a parse error on saving

**Automatic testing**

**Documentation**
